src/components/AnonymousChat/child/ChatItemTargetText.tsx

```diff
@@ -118,7 +119,7 @@ const ChatItemTargetText = ({
   return (
     <View style={[styles.chatContainer, isContinuous ? {marginTop: dimen.normalizeDimen(-4)} : {}]}>
       {renderAvatar()}
-      <ChatContextMenuView contextMenuType="TargetChatContextMenu" chat={chatItem}>
+      <ChatContextMenuView contextMenuType="TargetChatContextMenu" chat={chatItem} type={chatType}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `ChatContextMenuView` component now accepts a new prop `type`. Please ensure that this prop is properly validated and handled in the `ChatContextMenuView` component. If it's not required for all instances of `ChatContextMenuView`, consider making it optional to maintain backward compatibility.
```diff
- <ChatContextMenuView contextMenuType="TargetChatContextMenu" chat={chatItem} type={chatType}>
+ <ChatContextMenuView contextMenuType="TargetChatContextMenu" chat={chatItem} type={chatType || 'SIGNED'}>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->